.releaserc

```diff
@@ -25,24 +25,17 @@
         "changelogFile": "CHANGELOG.md"
       }
     ],
+    [
+      "@semantic-release/npm",
+      {
+        "npmPublish": false
```

Comment:
Don't publish to a registry, but we need this so that the `semantic-release/git` plugin will manage the `pacakge.json version` for us.